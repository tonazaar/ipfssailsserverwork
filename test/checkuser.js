
use ipfsmongo
show collections;
db.user.find({}).pretty();

