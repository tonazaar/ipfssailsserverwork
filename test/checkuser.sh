
mongo < checkuser.js  > checkuser.out
#does ever user has this

#list deviations

#2. tally with ipfs

#3. Does every record have this
