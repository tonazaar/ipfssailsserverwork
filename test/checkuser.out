MongoDB shell version v3.6.3
connecting to: mongodb://127.0.0.1:27017
MongoDB server version: 3.6.3
switched to db ipfsmongo
user
{
	"_id" : ObjectId("5e8df279c2340a51debb117a"),
	"createdAt" : 1586360953586,
	"updatedAt" : 1586360953586,
	"email" : "user1@dora.com",
	"username" : "user1",
	"userid" : "user1xxx",
	"role" : "user",
	"encryptedPassword" : "$2b$10$Ywz0.bYbubccf/oaFnk0huZcka3rQgoEdYVqp7EWI8Ys9mujDQjAK"
}
bye
